spring-cloud-starter-stream-processor-twitter-sentiment/src/test/java/org/springframework/cloud/stream/app/twitter/sentiment/processor/twitter/TwitterSentimentTensorflowProcessorIntegrationTests.java

```diff
@@ -51,6 +52,7 @@
 				"tensorflow.twitter.vocabulary=http://dl.bintray.com/big-data/generic/vocab.csv"
 		})
 @DirtiesContext
+@Ignore
```

Comment:
Well, in my comment there I suggested to add some message to the ignore.
Having that in logs or test report can make me to think one more time and unignore the test to check.

Thank you for a prompt turnaround anyway!